Obedient-Lobsters/BangazonAPI
# Added ProductOrder and Order models

# Description
Created ProductOrder Model for intersection table between Product #2  and Order #4  and created Order model #4 
 Related to #2  #4 
 ## Type of change


 # Testing Instructions
-Ensure that ProductOrder.cs and Order.cs are included in the Model folder

-In the directory BangazonAPI, use command "dotnet run" to run program. The server should then start and run without errors.

-Make sure all names line up with ERD.(https://drive.google.com/file/d/1Qo9O2oUezNd13Wsr0_VUbUtwwDo4-oAt/view)
 # Checklist:
